projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,6 +18,53 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const ConnectionItem = props => {
+	return props.connectionIds.map( connectionId => {
+		const connection = jetpackSocial.connections[ props.provider ][ connectionId ];
+		return (
+			<div>
+				<tr>
+					<th className={ styles.connectionRow }>Name</th>
+					<th className={ styles.connectionRow }>Image</th>
+				</tr>
+				<tr>
+					<td className={ styles.connectionRow }>{ connection.external_display }</td>
+					<td className={ styles.connectionRow }>
+						{ connection && (
+							<img
+								alt="connection_image"
+								src={ connection.profile_picture ? connection.profile_picture : null }
```

Comment:
You should be able to rely on optional chaining and nullish coalescing here.

```suggestion
								src={ connection?.profile_picture ?? null }
```